generatebundlefile/bundle.go

```diff
@@ -48,7 +48,7 @@ func NewBundleGenerate(bundleName string, opts ...BundleGenerateOpt) *api.Packag
 						Repository: "sample-Repository",
 						Versions: []api.SourceVersion{
 							{
-								Name:   "v0.0",
```

Comment:
I'm not entirely convinced we need to remove the v from our version here. Maybe I'm missing something though. We have `appVersion` which seems to be for humans and `version` for helm